midi/midi/decode.py

```diff
@@ -72,18 +79,27 @@ def __eq__(self, other):
                self.value == other.value and \
                self.use_velocities == other.use_velocities
 
+    def __repr__(self):
+        out_str = 'ActiveElement('
+        out_str += repr(self.height) + ', '
+        out_str += repr(self.value) + ', '
+        out_str += repr(self.use_velocities) + ')'
+
+        return out_str
```

Comment:
```suggestion
        return f'ActiveElement({self.height}, {self.value}, {self.use_velocities})'
```

Reply:
code shortened in f947ee90510cc06c7ead4eafe548daad3e73edbf